### DISCUSSION TITLE
[Feature] Scheduled Jobs

### DISCUSSION THREAD

**Discussion by ioslife** [immich-app/immich#4546] [Feature Request]
### The feature

The initial import is a CPU hog. It has my server running at 100% for the entire import.

I use my server as my DNS upstream as well as to run AdGuardHome and a few other important services behind. When Immich is stealing 100% of the CPU, the server becomes useless.

It would be nice to set a schedule for the jobs that are running instead of having them run on upload. I would like to specify "this job can run at 12AM."

This is only useful for the initial import, so this might be a silly request, but would be nice to have the option.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by alextran1502** [comment]
You can also limit the CPU of the microservices container as well

https://www.baeldung.com/ops/docker-memory-limit

**Reply by ioslife** [reply]
oh that is a good idea

**Reply by jrasm91** [reply]
you can also set job concurrency and/or pause background jobs entirely.
